gen/build_deploy/bash.py

```diff
@@ -136,9 +136,9 @@
 
 # check if sort -V works
 function check_sort_capability() {
-    $( echo '1' | sort -V >/dev/null 2>&1 || exit 1 )
+    $( echo '1' | sort -V >/dev/null 2>&1 )
     RC=$?
-    if [[ "$RC" -eq "2" ]]; then
+    if [[ "$RC" -eq "0" ]]; then
```

Comment:
It gets a little tricky now, because we are hiding this 'gnu sort specific exit code' check with-in this statement.

I think, the fix could be simpler and that's the reason the `||` logic is used in the first place.

```
$( echo 'a' | sort -V >/dev/null 2>&1 || exit 1 )
RC=$?
if [[ "$RC" -eq "1" ]]; then
```

* We are checking against the implanted value of 1, that is reached only when the command fails.